src/ops/transformations.jl

```diff
@@ -52,58 +37,14 @@ https://www.tensorflow.org/versions/r0.10/api_docs/python/array_ops.html#reshape
 @op Base.reshape(n::AbstractTensor, dims; name=nothing) =
   reshape(n, Tensor(Int32[dims...]); name = name)
 
-@op function Base.reshape(n::AbstractTensor, dims::AbstractTensor; name=nothing)
-    local desc
-    with_op_name(name, "Reshape") do
-        desc = NodeDescription("Reshape")
-        add_input(desc, n)
-        add_input(desc, dims)
-    end
-    Tensor(Operation(desc), 1)
+@op function Base.reshape(n::AbstractTensor, dims::AbstractTensor; kwargs...)
+    Ops.reshape(n, dims; kwargs...)
 end
 
 
 # if isdefined(Base, :slice)  # Removed in .6
```

Comment:
This is a breaking change for users of `slice` in 0.5,
since it was effectively exported before (by overloading `Base`)
and it is not exported now.

It is exported in 0.6
https://github.com/malmaud/TensorFlow.jl/blob/master/src/TensorFlow.jl#L128